genome.spec

```diff
@@ -75,7 +89,7 @@ typedef structure {
 } InferenceInfo;
 
 /*
-Structure for a single feature CDS
+Structure for a single coding sequence.
 
       flags are flag fields in GenBank format. This will be a controlled vocabulary.
         Initially Acceptable values are pseudo, ribosomal_slippage, and trans_splicing
```

Comment:
mRNA needs docs, can't comment any lower than this

Reply:
As above, I expanded the docs here, which has a lot of overlap with CDS. I didn't finish everything, but feel free to review what I have if you have time.